style.css

```diff
@@ -0,0 +1,631 @@
+* {
```

Comment:
- [ ] Let's try to render your typographies correctly. Right now they're rendering as `Arami`. The correct typography should be Cocogoose. You can see this in the last line of this image.

![image](https://user-images.githubusercontent.com/79114233/142569086-41ec54df-3c63-4942-aa78-6c70070d3627.png)

For reference, here it is your hero section:
![image](https://user-images.githubusercontent.com/79114233/142569220-3d7ea0fe-729a-4286-901f-e539b457f257.png)

Design:
![image](https://user-images.githubusercontent.com/79114233/142569267-0846bce9-fc3f-4671-b088-14dc08de6b76.png)